jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/FileInputsTests.java

```diff
@@ -74,7 +75,7 @@ public void smallChipsFileInputTest() {
         smallChipsFileInput.has().files(asList(pathTXT.getFileName().toString(), pathPNG.getFileName().toString()));
     }
 
-    @Test
+    @Test @Ignore
```

Comment:
Please update your branch with last changes from vuetify-develop

Reply:
It seems that there are no changes